src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorCodeLensProvider.ts

```diff
@@ -86,34 +85,11 @@ export class RazorCodeLensProvider
                 projection.position) as vscode.Location[];
 
             // We now have a list of references to show in the CodeLens.
-            // Let's remap any locations pointing to any background documents back to the original documents.
-            // Technically, Omnisharp should have already remapped those. We're just being extra cautious here.
-            const remappedLocations = new Array<vscode.Location>();
```

Comment:
We decided we didn't need to be extra cautious?

Reply:
Yeah I realized I was being silly as this will never happen. Our remap implementation will already ignore files that we can't remap. So this will never be used.